winui/Shadow/Customization.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Customization in WinUI Shadow control | Syncfusion
+description: Learn here all about the customization features such as color, opacity, position, and blur radius in Syncfusion WinUI Shadow control.
+platform: WinUI
+control: SfShadow
+documentation: ug
+---
+
+# Customization in WinUI Shadow
+
+This section explains the customization features available in the WinUI Shadow control.
+
+## Applying Color for shadow
+
+The shadow color can be customized using the `Color` property. The default value of `Color` property is `Black` color with 25% alpha value.
```

Comment:
The shadow color can be customized using the `Color` property. The default value of the `Color` property is `Black` color with a 25% alpha value.

Reply:
Changed the Color Property content as mentioned above.